src/api/tojs.ts

```diff
@@ -27,16 +27,17 @@ function toJSHelper(source, options: ToJSOptions, __alreadySeen: Map<any, any>)
     if (!options.recurseEverything && !isObservable(source)) return source
 
     if (typeof source !== "object") return source
+    
+    // Directly return null if source is null
+    if (source === null) return null
 
     // Directly return the Date object itself if contained in the observable
     if (source instanceof Date) return source
 
     if (isObservableValue(source)) return toJSHelper(source.get(), options!, __alreadySeen)
 
     // make sure we track the keys of the object
-    if (isObservable(source)) {
-        keys(source)
-    }
+    if (isObservable(source)) keys(source)
```

Comment:
removing brackets in if statements if not a healthy idea

Reply:
@ItamarShDev I know, but it seems that this project is following this pattern...

Reply:
OK. deleting the comment :)